.github/CODEOWNERS

```
@@ -1,76 +1,68 @@
 /layouts/partials/css.html @o3de/sig-docs-community-reviewers @o3de/sig-docs-community-site-reviewers
-/CONTRIBUTING @o3de/sig-docs-community-reviewers
+/static/images/blog/ @o3de/sig-docs-community-reviewers 
+/static/images/community/ @o3de/sig-docs-community-reviewers 
+/static/images/contribute/ @o3de/sig-docs-community-reviewers 
+/CONTRIBUTING.md @o3de/sig-docs-community-reviewers
 /README.md @o3de/sig-docs-community-reviewers
 
 # Release content
-/content/docs/release-notes/* @o3de/sig-docs-community-maintainers @o3de/sig-release-maintainers
-/static/images/release-notes/* @o3de/sig-docs-community-maintainers @o3de/sig-release-maintainers
+/content/docs/release-notes/ @o3de/sig-docs-community-maintainers @o3de/sig-release-maintainers
+/static/images/release-notes/ @o3de/sig-docs-community-maintainers @o3de/sig-release-maintainers
 
 # Docs content
-/content/docs/* @o3de/sig-docs-community-reviewers
 /content/smoketest.md @o3de/sig-docs-community-reviewers
-/static/docs/* @o3de/sig-docs-community-reviewers
+/static/docs/ @o3de/sig-docs-community-reviewers
+/static/files/ @o3de/sig-docs-community-reviewers
 /static/_redirects @o3de/sig-docs-community-reviewers
-/static/images/api-reference/* @o3de/sig-docs-community-reviewers
-/static/images/atom-guide/* @o3de/sig-docs-community-reviewers
-/static/images/community/* @o3de/sig-docs-community-reviewers
-/static/images/contribute/* @o3de/sig-docs-community-reviewers
-/static/images/contributing/* @o3de/sig-docs-community-reviewers
-/static/images/learning-guide/* @o3de/sig-docs-community-reviewers
-/static/images/shared/* @o3de/sig-docs-community-reviewers
-/static/images/tools-ui/* @o3de/sig-docs-community-reviewers
-/static/images/user-guide/* @o3de/sig-docs-community-reviewers
-/static/images/welcome-guide/* @o3de/sig-docs-community-reviewers
-/static/images/icons/* @o3de/sig-docs-community-reviewers
-/layouts/shortcodes/* @o3de/sig-docs-community-reviewers @o3de/sig-docs-community-site-reviewers
-/layouts/docs/* @o3de/sig-docs-community-reviewers @o3de/sig-docs-community-site-reviewers
-/layouts/partials/docs* @o3de/sig-docs-community-reviewers @o3de/sig-docs-community-site-reviewers
+/layouts/shortcodes/ @o3de/sig-docs-community-reviewers @o3de/sig-docs-community-site-reviewers
+/layouts/docs/ @o3de/sig-docs-community-reviewers @o3de/sig-docs-community-site-reviewers
+/layouts/partials/docs @o3de/sig-docs-community-reviewers @o3de/sig-docs-community-site-reviewers
 
-# Web content
-/assets/* @o3de/sig-docs-community-site-reviewers
-/layouts/* @o3de/sig-docs-community-site-reviewers
-/content/search/* @o3de/sig-docs-community-site-reviewers
-/static/js/* @o3de/sig-docs-community-site-reviewers
-/config.toml @o3de/sig-docs-community-site-reviewers
-/Makefile @o3de/sig-docs-community-site-reviewers
-/package.json @o3de/sig-docs-community-site-reviewers
+# Shared community and docs content
+/content/docs/contributing/ @o3de/sig-docs-community-reviewers
```

Suggest adding a rule specifically for contributing/to-code/, which allows code reviewers to also be maintainers of that section. 
```suggestion
/content/docs/contributing/ @o3de/sig-docs-community-reviewers
/content/docs/contributing/to-code/ @o3de/sig-docs-community-reviewers @o3de/reviewers
```

---

Great suggestion!